config/reports.json

```diff
@@ -428,6 +430,132 @@
 			"timeseries": {
 				"enabled": false
 			}
+		},
+		"workboxAdoption": {
+			"name": "Workbox Adoption",
+			"type": "URLs",
+			"downIsBad": true,
+			"description": "[Workbox](https://developers.google.com/web/tools/workbox/) is a set of libraries and Node.js modules that make it easy to cache assets and take full advantage of features used to build Progressive Web Apps. This report shows the number of PWA URLs that use them.",
+			"histogram": {
+				"minDate": "2016_01_01"
+			}
+		},
+		"medianLighthouseScore": {
+			"name": "Lighthouse Scores",
+			"type": "PWA Score",
+			"downIsBad": true,
+			"description": "[Lighthouse](https://developers.google.com/web/tools/lighthouse/) is an open-source, automated tool for improving the quality of web pages. This report shows the average Lighthouse PWA score.",
```

Comment:
s/This report/This metric/ here and throughout

Reply:
Addressed in https://github.com/HTTPArchive/httparchive.org/pull/51/commits/a29bbb01b5016998d334066a6b6849daacc27e65.